x-pack/plugins/fleet/common/constants/routes.ts

```diff
@@ -40,7 +40,7 @@ export const EPM_API_ROUTES = {
   DELETE_PATTERN: EPM_PACKAGES_ONE_WITH_OPTIONAL_VERSION,
   INSTALL_KIBANA_ASSETS_PATTERN: `${EPM_PACKAGES_ONE}/kibana_assets`,
   DELETE_KIBANA_ASSETS_PATTERN: `${EPM_PACKAGES_ONE}/kibana_assets`,
-  FILEPATH_PATTERN: `${EPM_PACKAGES_ONE}/{filePath*}`,
```

Comment:
It is expected to use a wildcard pattern filepath could be a multi segment parameter like for example `/api/fleet/epm/packages/nginx/1.24.0/img/logo_nginx.svg`

Reply:
IMO that's better communicated in the parameter description. For example, wherever the parameter is defined, add something like ```schema.string({ meta: { description: 'The file path, which can contain multiple directories and wildcards.' }, ...```.  Either way, the two places where this path parameter is defined need to match.

Reply:
@lcawl I think someone will have to fix the open API generator to make the parameter match similar to what has been done for `?` (optional parameter) https://github.com/elastic/kibana/blob/main/packages/kbn-router-to-openapispec/src/util.ts#L64-L70

Reply:
I've merged a commit that @jloleysens provided

Reply:
Thanks both of you, it looks good to me.